Sources/TuistGenerator/Generator/LinkGenerator.swift

```diff
@@ -169,7 +169,19 @@ final class LinkGenerator: LinkGenerating {
 
         try dependencies.forEach { dependency in
             if case let DependencyReference.absolute(path) = dependency {
-                precompiledFrameworkPaths.append(path)
+                if path.extension == "xcframework" {
```

Comment:
What about adding a unit test here that makes sure that the phase contains the build files for the xcframeworks?

Reply:
i am thinking of adding unit and integration tests in this PR.

Reply:
added